MapboxDirections/MBAnnotation.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+@objc(AnnotationType)
+public enum SegmentAttribute: Int, CustomStringConvertible {
```

Comment:
This type should be called simply `Attribute`, because `openStreetMapNodeIdentifier` is a node attribute, not a segment attribute.